src/.golangci.yml

```diff
@@ -0,0 +1,90 @@
+run:
```

Comment:
Shouldn't this rather be in the root directory? It would seem that `src` implies that there's source code there.

Reply:
The root directory is for the files of the registry (metadata for the providers/modules), src is where all the source code is.